cmds/rsdp/rsdp.go

```diff
@@ -52,15 +52,15 @@ func getRSDP(path string) (string, error) {
 	}()
 
 	var dataRead int
-	var exit int
+	var exit bool
 	dataRead = 1
-	exit = 0
-	for dataRead == 1 && exit == 0 {
+	exit = false
```

Comment:
this is unnecessary, exit is false by default

Reply:
thanks, I just updated the code